library/ws-widget/collection/src/lib/card-content-v2/card-content-v2.component.ts

```diff
@@ -381,8 +381,9 @@ export class CardContentV2Component extends WidgetBaseComponent
       })
     if(certificateData.issuedCertificates.length > 0) {
```

Comment:
please put additional check for certificateDate

Reply:
fixed